config/redirects

```diff
@@ -1,8 +1,8 @@
 define: base https://www.mongodb.com/docs/atlas/cli
-define: versions v1.0 v1.1 v1.2 v1.3 v1.4 v1.5 v1.6 v1.7 v1.8 v1.9 v1.10 v1.11 v1.12 v1.13 v1.14 v1.15 v1.16 v1.17 v1.18 v1.19 v1.20 v1.21 v1.22 v1.23 v1.24 v1.25 1.26 1.27 1.28 1.29 1.30 master
+define: versions v1.0 v1.1 v1.2 v1.3 v1.4 v1.5 v1.6 v1.7 v1.8 v1.9 v1.10 v1.11 v1.12 v1.13 v1.14 v1.15 v1.16 v1.17 v1.18 v1.19 v1.20 v1.21 v1.22 v1.23 v1.24 v1.25 1.26 1.27 1.28 1.29 1.30 1.31 master
```

Comment:
looks like we've missed adding the `v` before the version for the last few releases. We've also missed adding in the current version for the last few. adding that back in here and correcting the symlinks below
```suggestion
define: versions v1.0 v1.1 v1.2 v1.3 v1.4 v1.5 v1.6 v1.7 v1.8 v1.9 v1.10 v1.11 v1.12 v1.13 v1.14 v1.15 v1.16 v1.17 v1.18 v1.19 v1.20 v1.21 v1.22 v1.23 v1.24 v1.25 v1.26 v1.27 v1.28 v1.29 v1.30 v1.31 v1.32 master
```